slurm-advanced.tmpl

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
since `slurm-advanced.tmpl` is in `inst/` i don't think we need it in top-level (see [`check()`](https://github.com/databio/simpleCache/pull/5#issuecomment-318659752) results)